src/content/docs/errors-inbox/errors-inbox.mdx

```diff
@@ -49,7 +51,33 @@ Ready to get started? Make sure you have a [New Relic account](https://newrelic.
 
 Access to log management capabilities is limited by user type. See [user type capabilities](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type#user-type-capabilities).
 
-## Set up errors inbox [#get-started]
+## Use errors inbox directly from scoped views [#scoped]
+
+Scoped views allow you to view errors for entities in browser, mobile, and [Open Telemetry](https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction/#benefits). Access the errors inbox view for a specific entity by navigating to that entity in the browser or mobile monitoring view and selecting **Errors inbox** in the left nav. 
```

Comment:
Perhaps start with:
"[Entity](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/what-entity-new-relic/) scoped views allow you to view the errors for a single APM, Browser, Mobile, or [Open Telemetry]() application."

Consider adding "... in the left nav under **Triage**"